drivers/mem_manage/mm_drv_intel_adsp_tlb.c

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (c) 2021 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <spinlock.h>
+#include <sys/__assert.h>
+#include <sys/check.h>
+#include <sys/mem_manage.h>
+#include <sys/util.h>
+
+#include <soc.h>
+#include <cavs-mem.h>
+
+#include "mm_drv_common.h"
+
+#define TLB_BASE 0x3000
```

Comment:
This should come from devicetree

Reply:
Updated to use devicetree.

However, this doesn't expose APIs as in, for example, GPIO drivers, where there are driver pointers. This is modeled similar to timer driver where there can only be one instance. Let me know where I can put it, if not in `drivers/`.